docs/support/support_overview.md

```diff
@@ -0,0 +1,91 @@
+<!--
+  ~ Copyright (c) 2025 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Arista AVD A-Care TAC Support Overview
+
+Arista AVD is a multi-domain network automation framework consumable as code, providing a continuous design framework. The AVD project is offered as an open-source project and backed by the world-class Arista Technical Support team through A-Care Service. TAC support for AVD must be purchased separately; see the ordering information below.
```

Comment:
Arista AVD is a network automation framework covering multiple network design domains. AVD is an open-source project maintained by a dedicated Arista engineering team. Arista AVD customers have an option of ordering TAC support through A-Care Service. TAC support AVD must be purchased separately; see the ordering information below. Without support contract AVD code can be used on as-is basis, without any warranty and with best effort problem resolution via GitHub issues.

Not insisting on this description, but I'd certainly change the starting line.

Reply:
I agree with Petr, plainer language is better. Just minor corrections:

Arista AVD is a network automation framework covering multiple network design domains. AVD is an open-source project maintained by a dedicated Arista engineering team. Customers have the option of purchasing TAC support for AVD through the A-Care Service. TAC support for AVD must be purchased separately; see the ordering information below. Without a support contract AVD code can be used as-is, without any warranty and with best effort problem resolution via GitHub issues.

Reply:
Thanks for the recommendation. I rephrased it!